sem-version

```diff
@@ -67,6 +67,21 @@ version::change_python() {
 
   return $?
 }
+
+################################################################################
+# Added by Mihalis Tsoukalos
+################################################################################
+version::change_c() {
+  version::change "change-gcc-version"
+
+  return $?
+}
+
+version::change_cpp() {
+  version::change "change-gcc-version"
```

Comment:
is it `change-gcc-version` for C++ too?

Reply:
Yes - because both C and C++ are using the same compiler.